editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import PanelBody from 'components/panel/body';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+function PostStatus() {
+	return (
+		<PanelBody toggle={ __( 'Status & Visibility' ) }>
+			<div>
+				<div className="editor-sidebar-post-status__row">
+					<span>{ __( 'Pending review' ) }</span>
+					<FormToggle />
```

Comment:
This causes a warning to be logged about `button` as descendant of `button` (`<FormToggle>` of `<PanelBodyToggle>`)

```
react-dom.development.js:498 Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    in button (created by Button)
    in Button (created by IconButton)
    in IconButton (created by PanelBodyToggle)
    in button (created by PanelBodyToggle)
    in PanelBodyToggle (created by PanelBody)
    in div (created by PanelBody)
    in PanelBody (created by PostStatus)
    in PostStatus (created by PostSettings)
    in div (created by Panel)
    in Panel (created by PostSettings)
    in PostSettings (created by Connect(PostSettings))
    in Connect(PostSettings) (created by Sidebar)
    in div (created by Sidebar)
    in Sidebar (created by Layout)
    in div (created by Layout)
    in Layout (created by Connect(Layout))
    in Connect(Layout)
    in Provider
    in Provider
```

I'm finding buttons to be rather difficult to work with due to restrictions on the types of of elements they accept as children.